skimage/registration/masked_register_translation.py

```diff
@@ -62,33 +58,37 @@ def masked_register_translation(
     .. [2] D. Padfield. "Masked FFT registration". In Proc. Computer Vision and
            Pattern Recognition, pp. 2918-2925 (2010).
            :DOI:`10.1109/CVPR.2010.5540032`
+
     """
-    if target_mask is None:
-        target_mask = np.array(src_mask, dtype=np.bool, copy=True)
+    if moving_mask is None:
+        moving_mask = np.array(reference_mask, dtype=np.bool, copy=True)

```

Comment:
```suggestion
        moving_mask = reference_mask.astype(np.bool, copy=True)

```

Reply:
Come to think of it, is the mask modified? Maybe the copy is unnecessary here?

Reply:
I think it is not...